benchmarks/serving/backend_request_func.py

```diff
@@ -0,0 +1,277 @@
+import json
+import os
+import time
+from typing import Dict, Union
+
+import aiohttp
+
+AIOHTTP_TIMEOUT = aiohttp.ClientTimeout(total=6 * 60 * 60)
+
+
+async def async_request_tgi(
+    prompt: str,
+    api_url: str,
+    prompt_len: int,
+    output_len: int,
+    best_of: int,
+    use_beam_search: bool,
+    **kwargs,
+) -> Dict[str, Union[str, bool, float]]:
```

Comment:
can we turn both the input and output into data classes so it's easier to align the interface among the options? btw, this list of functions are great!

Reply:
Done!